config/config-es.json

```diff
@@ -910,7 +920,8 @@
         "titleUrlSafeText": "Sitio seguro según IA (probabilidad <PERCENTAGE>%)",
         "reasonText": "Motivo: <REASON>",
         "buttonBadDecisionText": "Mala decisión",
-        "checkboxDonotShowText": "No volver a mostrar esto"
+        "checkboxDonotShowText": "No volver a mostrar esto",
+        "progressInfoText": "Here is the progress while you are teaching Focus Bear to keep allowing sites/apps but within a few days, you'll have taught Focus Bear what apps and websites are productive.[NEEDSTRANSLATION]"
```

Comment:
You're teaching Focus Bear what sites and apps are productive for you. It takes a bit of effort at first but within a few days, you'll have taught Focus Bear what apps and websites are productive.